packages/clay-shared/src/__tests__/FocusScope.tsx

```diff
@@ -108,10 +168,185 @@ describe('FocusScope', () => {
+		const htmlButton1 = document.getElementById('button1') as HTMLElement;
+		const htmlButton2 = document.getElementById('button2') as HTMLElement;
+
+		const reactButton = document.getElementById(
+			'reactButton'
+		) as HTMLElement;
+
+		htmlButton2.focus();
+
+		expect(htmlButton2).toHaveFocus();
+
+		userEvent.tab({shift: true});
+
+		expect(reactButton).toHaveFocus();
+
+		userEvent.tab({shift: true});
+
+		expect(htmlButton1).toHaveFocus();
+	});
+
+	it('interacts with down arrow key', () => {
+		const linkInPortalOne = document.getElementById(
+			'linkInPortal1'
+		) as HTMLElement;
+		const linkInPortalTwo = document.getElementById(
+			'linkInPortal2'
+		) as HTMLElement;
+		const linkInPortalThree = document.getElementById(
+			'linkInPortal3'
+		) as HTMLElement;
+
+		linkInPortalOne.focus();
+
+		expect(linkInPortalOne).toHaveFocus();
+
+		fireEvent.keyDown(linkInPortalOne, {key: 'ArrowDown'});
+
+		expect(linkInPortalTwo).toHaveFocus();
+
+		fireEvent.keyDown(linkInPortalTwo, {key: 'ArrowDown'});
+
+		expect(linkInPortalThree).toHaveFocus();
+
+		fireEvent.keyDown(linkInPortalThree, {key: 'ArrowDown'});
+	});
+
+	it('interacts with up arrow key', () => {
+		const linkInPortalOne = document.getElementById(
+			'linkInPortal1'
+		) as HTMLElement;
+		const linkInPortalTwo = document.getElementById(
+			'linkInPortal2'
+		) as HTMLElement;
+		const linkInPortalThree = document.getElementById(
+			'linkInPortal3'
+		) as HTMLElement;
+
+		linkInPortalThree.focus();
+
+		expect(linkInPortalThree).toHaveFocus();
+
+		fireEvent.keyDown(linkInPortalThree, {key: 'ArrowUp'});
+
+		expect(linkInPortalTwo).toHaveFocus();
+
+		fireEvent.keyDown(linkInPortalTwo, {key: 'ArrowUp'});
+
+		expect(linkInPortalOne).toHaveFocus();
+
+		fireEvent.keyDown(linkInPortalOne, {key: 'ArrowUp'});
+	});
+
+	describe('FocusScope outside the React Tree', () => {
+		beforeEach(() => {
+			jest.clearAllTimers();
+			document.body.innerHTML = '';
+			cleanup();
+		});
+
+		it('interacts with React.Portal', () => {
+			render(
+				<>
+					<ClayPortal>
+						<div id="content">
+							<button id="button1" />
+						</div>
+					</ClayPortal>
+
+					<DropDownWithState>
+						<ClayDropDown.ItemList>
+							{[
+								{href: '#one', label: 'one'},
+								{href: '#two', label: 'two'},
+								{href: '#three', label: 'three'},
+							].map((item, i) => (
+								<ClayDropDown.Item
+									href={item.href}
+									key={i}
+									spritemap="/foo/bar"
+								>
+									{item.label}
+								</ClayDropDown.Item>
+							))}
+						</ClayDropDown.ItemList>
+					</DropDownWithState>
+				</>
+			);
+			const dropdownButton = document.querySelector(
+				'.dropdown-toggle'
+			) as HTMLElement;
+			const htmlButton1 = document.getElementById(
+				'button1'
+			) as HTMLElement;
+
+			dropdownButton.click();
+
+			userEvent.tab();
+
+			expect(dropdownButton).toHaveFocus();
+
+			userEvent.tab();
+
+			expect(htmlButton1).toHaveFocus();
```

Comment:
Hey @sergiojimcos I think this behavior is wrong, it should focus on DropDown links and then focus on `button1`, I think we found an edge case here 😅, I did a test and it seems that I can only reproduce it the first time then it works normally.

I would say switch to this expected behavior and add a `skip` to this test so it doesn't crash this PR and then we work on fixing that behavior and remove the `skip` in other PR.

Reply:
Could this be because opening DropDown and pressing tab close the dropdown? I think if this happens it makes sense but it was also to focus on `button1` first ah unless the focus is on the button toggle due to the click.

Reply:
Hey @matuzalemsteles! It makes sense because I was testing it manually and the focus closes the Dropdown. I'm going to investigate a bit more about this. If I don't find any fix, I'll update it to `skip` this test and check it later in another PR.